src/less-style/modals.less

```diff
@@ -37,7 +37,7 @@
 }
 
 // Full-screen modal for displaying form XML
-.fd-source-modal {
+.fd-full-screen-modal {
```

Comment:
I removed those. I initially had them separately and hesitated to remove something descriptive